pkg/agent/openflow/network_policy.go

```diff
@@ -738,6 +739,14 @@ func (c *client) InstallPolicyRuleFlows(ruleID uint32, rule *types.PolicyRule, n
 	if err := c.applyConjunctiveMatchFlows(ctxChanges); err != nil {
 		return err
 	}
+
+	// Count up antrea_agent_ingress_networkpolicy_rule or antrea_agent_egress_networkpolicy_rule
```

Comment:
Why not installOFRule function in pkg/agent/controller/networkpolicy/reconciler.go?

This function could be used to track number of network policies in addition to the current metric, number of network policy rules.

Reply:
Thank you for the suggestion, it sounds great. I'll take a look into it.